wagtail_localize/views/edit_translation.py

```diff
@@ -268,15 +269,46 @@ def widget_from_field(field):
             if issubclass(field.related_model, Page):
                 edit_handler = tab_helper.get_field_edit_handler(field.name)
 
+                if WAGTAIL_VERSION >= (2, 17):
+                    # @see https://github.com/wagtail/wagtail/pull/7684
+                    # the target_models is set in the ModelFieldRegistry for ForeignKeys
+                    from wagtail.admin.forms.models import registry
+
+                    allowed_page_types = [
```

Comment:
Could we move this to an `else:` clause of the `if` statement just below?

Reply:
good shout 👍🏼 sorted in [d9dc215](https://github.com/wagtail/wagtail-localize/pull/536/commits/d9dc215761754067b95751b566e3ec2c98b5f603)